tag_manager/content_security_policy.rst

```diff
@@ -177,7 +174,7 @@ Following example configuration of CSP assumes:
 .. code-block:: text
 
     Content-Security-Policy: default-src 'self';
```

Comment:
```suggestion
    Content-Security-Policy: default-src 'none';
```

Reply:
Agreed, however I am thinking about some additional comment, that this is recommended (generally, not only by us), not required. Without it, it might raise question, if this can be set to anything else.

Reply:
I'd add none and nothing else, as you said yesterday - someone who is configuring this already knows something about the csp

Reply:
Done.

Reply:
Added `https` everywhere.